src/js/App.ts

```diff
@@ -1,66 +1,51 @@
-import ManageItemView from './views/mangeItemView';
-import ChargeMoneyView from './views/chargeMoneyView';
-import PurchaseItemView from './views/purchaseItemView';
-import VendingMachine from './vendingMachine/vendingMachine';
-import AppView from './views/AppView';
+import MainView from './views/main';
 import { URL, CUSTOM_EVENT } from './constants/appContants';
-import { SELECTOR } from './constants/viewConstants';
 import { RouteChangeDetailType } from './types/types';
+import SignView from './views/sign';
 
 export default class App {
-  appView: AppView;
-  vendingMachine: VendingMachine;
-  manageItemView: ManageItemView;
-  chargeMoneyView: ChargeMoneyView;
-  purchaseItemView: PurchaseItemView;
+  mainView: MainView;
+  signView: SignView;
 
   constructor() {
-    this.appView = new AppView();
-    this.vendingMachine = new VendingMachine();
-    this.manageItemView = new ManageItemView(this.vendingMachine);
-    this.chargeMoneyView = new ChargeMoneyView(this.vendingMachine);
-    this.purchaseItemView = new PurchaseItemView(this.vendingMachine);
+    this.mainView = new MainView();
+    this.signView = new SignView();
```

Comment:
```suggestion
  constructor(private mainView: MainView, private signView: SignView) {
```

이렇게 초기화하면 자동으로 this에 생성자로 받은 인자들이 할당돼요~
[참고](https://prolog.techcourse.co.kr/studylogs/2165)